lldb/test/API/lang/swift/nested_generic_class/TestSwiftNestedGenericClass.py

```diff
@@ -0,0 +1,17 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import unittest2
+
+
+class SwiftNestedGenericClassTest(TestBase):
+
+    @swiftTest
+    def test(self):
+        """Tests that a generic class type nested inside another generic class can be resolved correctly from the instance metadata"""
+        self.build()
+        (target, process, thread, breakpoint) = lldbutil.run_to_source_breakpoint(self, 
+                "break here", lldb.SBFileSpec("main.swift"))
```

Comment:
minor note: since the vars (`target`, etc) aren't being used you could not assign any of them. Ex:

```suggestion
        lldbutil.run_to_source_breakpoint(self, "break here", lldb.SBFileSpec("main.swift"))
```